frontside/lib/main.dart

```diff
@@ -16,8 +16,9 @@ class Sk8 extends StatelessWidget {
       theme: defaultTheme,
       routeInformationParser: BeamerParser(),
       routerDelegate: BeamerRouter.routerDelegate,
-      backButtonDispatcher:
-          BeamerBackButtonDispatcher(delegate: BeamerRouter.routerDelegate),
+      //https://pub.dev/documentation/beamer/latest/beamer/BeamerBackButtonDispatcher-class.html
+      //Why not stay in default button behavior?
+      backButtonDispatcher: BeamerBackButtonDispatcher(delegate: BeamerRouter.routerDelegate),
```

Comment:
I think it's required to provide the same behavior to the Android back button:
> NOTE that Beamer can integrate Android's back button to do beamBack if possible when all the pages from current BeamLocation have been popped. This is achieved by setting a back button dispatcher in MaterialApp.router.

Source: https://pub.dev/packages/beamer#beaming-back